xapian-applications/omega/.gitignore

```diff
@@ -39,6 +39,7 @@
 /md5test
 /urlenctest
 /utf8converttest
+/omindexcheck
```

Comment:
The other test programs are listed in sorted order here...

Reply:
Sorry, I didn't notice it. I will sort it.